docs/user-guide/admin-cli.md

```diff
@@ -653,17 +698,112 @@ enhancement tracked [here](https://github.com/vmware/docker-volume-vsphere/issue
 ```
+
+## Config
+
+Creates, removes, moves and reports on status of config DB. Config DB keep authentication information, and without initializing it no access control is supported. Also, configuring access control will fail, e.g.
+```
+[root@localhost:~] vmdkops_admin  vm-group create --name MY
+Internal Error(Error: Please init configuration in vmdkops_admin before trying to change it)
+```
+
+#### Init
+
+Initialized the config is optional. If the config is not initialized, there will be no access control and all `vm-group` commands will fail with appropriate messages.
+
+Before configuring access control or quotas, the config needs to be inited to either local (`init --local`) or shared (`init --datastore=ds_name) mode
+
+```
+[root@localhost:~] vmdkops_admin config init -h
+usage: vmdkops_admin.py config init [-h] [--local] [--force]
+                                    [--datastore DATASTORE]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --local               Allows local (SingleNode) Init
+  --force               Force operation, ignore warnings
+  --datastore DATASTORE
+                        Config DB will be placed on a shared datastore
+```
+
+Example:
+
+```
+[root@localhost:~] vmdkops_admin status
+Version: 0.12.fea683a-0.0.1
+Status: Running
+DB_LocalPath: /etc/vmware/vmdkops/auth-db
+DB_SharedLocation: n/a
+DB_Mode: NotConfigured (no local DB or symlink)  <===== NOT CONFIGURED
+Pid: 5979199
+Port: 1019
+LogConfigFile: /etc/vmware/vmdkops/log_config.json
+LogFile: /var/log/vmware/vmdk_ops.log
+LogLevel: DEBUG
+
+[root@localhost:~] vmdkops_admin config init --local
+Creating new DB at /etc/vmware/vmdkops/auth-db
+Restarting the vmdkops service to pick up new configuration
+Stopping vmdkops-opsd with PID=5979199
+vmdkops-opsd is not running
+Starting vmdkops-opsd
+vmdkops-opsd is running pid=5979684
+
+[root@localhost:~] vmdkops_admin status
+Version: 0.12.fea683a-0.0.1
+Status: Running
+DB_LocalPath: /etc/vmware/vmdkops/auth-db
+DB_SharedLocation: /etc/vmware/vmdkops/auth-db
+DB_Mode: SingleNode (local DB exists)    <==== LOCAL configuration
+Pid: 5979684
+Port: 1019
+LogConfigFile: /etc/vmware/vmdkops/log_config.json
+LogFile: /var/log/vmware/vmdk_ops.log
+LogLevel: DEBUG
+```
+
+
+#### Remove
+
+Allows to remove configuration. Either `--local` or `--datastore` flag needs to be passed.
+Also to make sure it is not typed accidentally, the command requires `--force` flag to actually execute the request
+
+```
+[root@localhost:~] vmdkops_admin config remove -h
+usage: vmdkops_admin.py config remove [-h] [--local] [--no-backup] [--force]
+                                      [--datastore DATASTORE]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --local               Removes only local link or local DB
+  --no-backup           Do not create DB backup before removing
+  --force               Force operation, ignore warnings
+  --datastore DATASTORE
```

Comment:
For 2 reasons. Make admin stop and think, and allow removal of orphan shared DBs. 

And yes, they can say ''config remove --local"